packages/patternfly-3/patternfly-react-extensions/less/select.less

```diff
@@ -0,0 +1,141 @@
+@select-max-height: 205px;
+@select-min-width: 220px;
+
+.select-container {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
+  padding-right: 0;
+  overflow: hidden;
+  width: 100%;
+  min-width: @select-min-width;
+  border: 1px solid #ccc;
+  z-index: 1000;
+
+  &.open {
+    box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+  }
+
+  .select-dropdown-toggle {
+    display: flex;
+    align-items: center;
+    text-align: left;
+    border: 0;
+
+    .select-toggle-placeholder {
+      font-style: italic;
+      color: #9c9c9c;
+    }
+
+    .select-caret {
+      margin-left: auto;
+    }
+  }
+
+  .select-body-container {
+    display: flex;
+    flex-direction: column;
+    padding-left: 0;
+    padding-right: 0;
+    max-height: @select-max-height;
+    overflow: hidden;
+    width: 100%;
+    min-width: @select-min-width;
+    z-index: 1000;
+
+    .select-input-search {
+      position: relative;
+
+      input {
+        width: 100% !important;
```

Comment:
this is a `form-control` which already has width: 100%. Why this and why the `!important`?